examples/ibm-transit-gateway/README.md

```diff
@@ -0,0 +1,66 @@
+# Example for Direct Link resources
+
+This example shows how to create Direct Link resources.
```

Comment:
why the reference to "Direct Link"?

Reply:
Hi Frederic.. Thanks, Yes, it should be changed, no need of "Direct Link" reference, will change it.